### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ayalash**
This PR contains 3 commits:
1. Drop support for python version < 3.6
2. Update of README.rst links
3. Use importlib instead of imp, because the latest is deprecated

I dropped support for old python versions because the new importlib is not compatible with them and imp is deprecated with the new versions.

### CODE REVIEW

**Comment by ayalash** [comment]
@vmalloc I'm not deeply familiar with either imp or importlib - I changed the code according to "google". The unittest passed, but if I wrote something wrong then sorry

**Comment by vmalloc** [comment]
@ayalash as long as it passes the integration tests and assertions get properly rewritten with it (i.e. Slash runs fine and formats assertions properly) I'm ok with it.

**Review by vmalloc — APPROVED** [review]
LGTM

**Comment by ayalash** [comment]
@vmalloc I run the unittests of emport, dessert (with the new linked PR version) and slash with this version, everything passed.
I run a small "real" slash test that fails on assertion and it looked ok as well.

If it is enough in your opinion, please merge me both here and in dessert (I don't have merge permissions because it is under vmalloc user).

**Comment by ayalash** [comment]
Please notice that once this PR is merged, the https://github.com/vmalloc/emport/pull/7 is obsolete.

**Comment by vmalloc** [comment]
Merged. Thanks!

**Pull request merged**
